Makefile

```
@@ -1156,61 +1156,61 @@ REF_MANUAL_ATTRIBUTES = \
   --attribute UNICODE_SOURCE=$(UNICODE_SOURCE)
 
 $(BUILD_DIR)/generated/doc/unicode_version.adoc:
-	mkdir -p $(@D)
-	cd $(FZ_SRC) && git log modules/base/src/encodings/unicode/data.fz  | grep -E "^Date:" | head | sed "s-Date:   -:UNICODE_VERSION: -g" | head -n1 > $(realpath $(@D))/unicode_version.adoc
+	mkdir --parents $(@D)
+	cd $(FZ_SRC) && git log modules/base/src/encodings/unicode/data.fz  | grep --extended-regexp "^Date:" | head | sed "sate:   -:UNICODE_VERSION: -g" | head -n1 > $(realpath $(@D))/unicode_version.adoc
```

```suggestion
	cd $(FZ_SRC) && git log modules/base/src/encodings/unicode/data.fz  | grep --extended-regexp "^Date:" | head | sed "s-Date:   -:UNICODE_VERSION: -g" | head -n1 > $(realpath $(@D))/unicode_version.adoc
```

This is not an option